extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -805,6 +811,15 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
           this.getAmpDoc(), 'amp-analytics');
     }
 
+    const nameframeExperimentConfig = responseHeaders.get('amp-nameframe-exp');
```

Comment:
In the future, you don't need to create your own header, instead you can just use postAdResponseExperimentFeatures in amp-a4a.js.  Can discuss offline how it gets populated,.

Reply:
Will do moving forward. Are we okay to keep this as is for this experiment?

Reply:
Yes that's fine